settings.gradle

```diff
@@ -4,4 +4,5 @@ include 'embulk-core'
 include 'embulk-standards'
 include 'embulk-cli'
 include 'embulk-test'
+include 'embulk-jruby-strptime'
```

Comment:
This doesn't include `strftime` but, I renamed several classes. `StrftimeToken` to `StrptimeToken`, `StrftimeFormat` to `StrptimeFormat`. a4b40869b17eb29dfaa01dfb72998f4a45dae188